client/rest-high-level/src/test/java/org/elasticsearch/client/MachineLearningIT.java

```diff
@@ -225,6 +226,7 @@
 
     @After
     public void cleanUp() throws IOException {
+        ensureNoInitializingShards();
```

Comment:
I added this as I didn't see any checks in our cleanup code for this. It just seems prudent.